src/components/Header.js

```diff
@@ -1,7 +1,3 @@
 export default function Header() {
-	return (
-		<header>
-			<nav></nav>
-		</header>
-	);
+	return <header></header>;
```

Comment:
please remove empty components - because I am super mean 😁

Reply:
yes, you are! ;-)